src/agents/workflows/subgraphs/step2_router/graph.py

```
@@ -31,20 +32,183 @@
+            canonical = _normalize_feature_name(step)
+            if canonical and canonical not in plan:
+                plan.append(canonical)
+    return plan
+
+
+def intent(state: AgentState) -> AgentState:
+    """사용자 질문의 의도를 파악하고, RAG 호출 필요 여부 등을 결정합니다.
+    어떤 경우든 router 그래프는 여기서 종료되고, maingraph가 다음을 결정합니다.
+    """
+    print("---ROUTER: INTENT DETECTION---")
+    latest_question, ocr_full_text, combined_question = _collect_user_context(state)
```

Ruff (0.14.13)

Unpacked variable `latest_question` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)